google/ggrc-core
# QUICK-FIX Lint python files

# Issue description

This PR should resolve all pep8 errors in our codebase, except the old migration files that we do not change due to our migration policy. For that reason, the old migration files are ignored by flake8 tests.

# Steps to test the changes

Basic sanity check of the app, and check that all files truly pass flake8 checks.
Try to find what code was modified and if there you can produce a regression with it.

# Solution description

Lint bad python files, and ignore old python migration files.

Note: I have also pushed an some flake8 errors to this PR and the tests still worked. After the code style failed, I reverted those PRs.

# Sanity checklist




# PR Review checklist
